pkg/node/requester.go

```diff
@@ -53,6 +53,11 @@ type Requester struct {
 	debugInfoProviders []models.DebugInfoProvider
 }
 
+type MetaStore interface {
+	ReadInstallationID() (string, error)
+	ReadInstanceID() (string, error)
+}
+
```

Comment:
why introducing an interface instead of just passing the actual metastore to `NewRequesterNode`?

Reply:
To prevent the receivers of the type from calling methods that can modify the contents of the store - this is a read only store given this interface. It will also make testing easier in the future.